lib/note-content-editor.tsx

```diff
@@ -731,11 +732,25 @@ class NoteContentEditor extends Component<Props> {
   };
 
   render() {
-    const { fontSize, noteId, theme } = this.props;
+    const { fontSize, lineLength, noteId, theme } = this.props;
     const { content, editor, overTodo } = this.state;
 
+    let editorPadding = 25;
+
+    if (lineLength === 'narrow' && this.contentDiv?.current) {
+      const width = this.contentDiv?.current?.offsetWidth;
```

Comment:
this is the line that will lose a `?` once `contentDiv` is defined at the top of the class

```ts
const width = this.contentDiv.current?.offsetWidth;
```